qemu/tests/cfg/hotplug_unplug_during_io_repeat.cfg

```diff
@@ -0,0 +1,27 @@
+- hotplug_unplug_during_io_repeat:
+    no RHEL.3.9
+    no ide
+    virt_test_type = qemu
+    type = hotplug_unplug_during_io_repeat
+    images += " stg0"
+    boot_drive_stg0 = no
+    image_name_stg0 = images/storage0
+    image_size_stg0 = 30G
+    remove_image_stg0 = yes
+    force_create_image_stg0 = yes
+    kill_vm_on_error = yes
+    iozone_timeout = 7200
+    sleep_time = 30
+    repeat_time = 300
```

Comment:
hi yhong, discussed with feature owners,  repeat_time = 100 is ok.

Reply:
ok , have updated it, thanks.